Laptop Device

tylercraig9332 opened the pull request:
#19 
So, in SystemSelection.jsx I decided to use a local variable for the device types so they can be displayed in the correct order, but my concern is where does the prop, jsonfilters, come from outside the context of the file. I'm still pretty new to React so I'm not sure if what I did was okay or do I need to remove another call/pass to that prop somewhere else.

Stardog commented:
Good work so far. There are some additional changes I would like you to make.
Let's differentiate pcs from laptops. Make a new table systems_laptop table. The updated tables should look like this:

systems_laptop
id, device_id, os, system_usage, rotation, check_in, docking_stand

systems_pc will have the same columns BUT remove is_laptop and docking_stand.

Also, if you can think of a clever upgrad path that would transfer laptops over to the new table, that would be excellent.

tylercraig9332 commented:
# Current issues and Progress:
* ~~Can't delete Laptop device when it's in Surplus.~~
    * Fixed in commit https://github.com/tylercraig9332/systemsinventory/commit/93cc9589c8bdcbe6a6811ebda73f44708b2de4d0.
* ~~Edit on laptop doesn't work~~
    * Fixed in commit 7117f26
* ~~I need to clean up the extra fields laying around that are a part of laptop~~
    * The server fields...
    * Also need to add back in Docking Stand (Station) 
   * Addressed in commit 7117f26
* ~~Need to finish the sql in the boost update that transfers laptops from systems_pc to systems_laptop~~
    * Not worrying about this.
* ~~Add back 2nd mac address for PCs~~
* ~~Make monitor size for laptop only appear when Docking Station is clicked~~
